Add issue templates (#75)
meian/gcf: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature request
about: Request for your favorite features.
title: "[Feature Request]"
labels: enhancement
assignees: ''

---

### The feature you want to use.

Describe the feature name and how feature processes.

> ### What benefit with this feature?

Author: ```suggestion
### What benefit with the feature?
```

直し忘れ

Describe a clear and concise what achive from the feature.  
We also welcome to post  how to use example code.

> ### Is possible achive by other features in project or its combination to alternatives?

Author: ```suggestion
### Is possible achive alternative by other features in project or its combination?
```
こっちの方が訳がしっくり来た

> Describe whether you can achive you want with this feature in already implemented another features.  

Author: ```suggestion
Describe whether you can achive you want in already implemented another features.  
```
余計な単語がついてた
